### ISSUE TITLE
 Learningimage-to-imagetranslationusingpairedandunpairedtrainingsamples

### ISSUE THREAD

**Issue by kiyaaabdian** [shivom9713/Thermal2RGB#1]
hello
I have some questions about semi supervised cyclegan, can you help me?
I have 75000 images that 13000 of them are paired and 62000 of them are unpaired.
 1-how many epoches we need to train the network? and I don't know about the numbers of epoches for supervised section?
2-do we need validation set? 
3-we have 4 rtx3090 gpu for training, what does the best batch size for this condition?
